js/historian.js

```diff
@@ -344,6 +344,22 @@ historian.singleView_ = [
     defaultLevelMetricOverride: '',  // Disable default level metric.
     defaultXExtentLogs: [],  // Fit any shown data.
     showReportTaken: true
+  },
+  {
```

Comment:
Will this show up even if it is an old bugreport

Reply:
No, the sensor historian tab will not show up if the sdkversion < 26.